.github/workflows/workflow.yaml

```diff
@@ -12,19 +12,19 @@ on:
 jobs:
   validate:
     name: Validate
-    runs-on: [self-hosted, linux, ARM64]
+    runs-on: ubuntu-latest
     steps:
     - name: Check out code
       uses: actions/checkout@v1
 
     - name: Run a Terraform init
-      uses: docker://hashicorp/terraform:0.12.11
+      uses: docker://hashicorp/terraform:0.14.7
       with:
         entrypoint: terraform
         args: init
 
     - name: Run a Terraform validate
-      uses: docker://hashicorp/terraform:0.12.11
```

Comment:
Try setting the `AWS_DEFAULT_REGION` variable here as shown here: https://docs.github.com/en/actions/learn-github-actions/environment-variables#using-contexts-to-access-environment-variable-values to see if this would work